pkg/notifiers/mailer.go

```diff
@@ -30,6 +32,17 @@ type Mailer struct {
 	SMTP
 }
 
+func getMessage(filters *models.Filters) string {
+	msg := "Vaccines are available in the following centers :"
+	if filters != nil {
+		msg = fmt.Sprintf("Vaccines are available in the following centers for %d+ from %s", filters.Age, time.Now().Format("02-01-2006"))
```

Comment:
Lets move this identification of default date if user has not provided in the command itself instead of replacing the date with default date across. We already have this duplicated in two places

1. Here in the message
2. When the query string is appended to the cowin url

Reply:
I have put check if it is empty then will have default date as today. So no need to push current date  again.